crates/astria-sequencer/Cargo.toml

```diff
@@ -6,9 +6,11 @@ rust-version = "1.70.0"
 
 [dependencies]
 astria-proto = { path = "../astria-proto" }
+astria-sequencer-validation = { path = "../astria-sequencer-validation" }
 telemetry = { package = "astria-telemetry", path = "../astria-telemetry" }
 
 anyhow = "1"
+digest = "0.10"
```

Comment:
remove? It's only used in the validation crate?

Reply:
nice catch!